x/model/types/errors.go

```diff
@@ -102,7 +102,7 @@ func NewErrModelVersionAlreadyExists(vid interface{}, pid interface{}, softwareV
 func NewErrOtaURLCannotBeSet(vid interface{}, pid interface{}, softwareVersion interface{}) error {
```

Comment:
It seems this function is not used now and can be deleted.

Reply:
Deleted